src/main/java/com/github/alathra/siegeengines/SiegeEngine.java

```diff
@@ -34,55 +34,55 @@
 public class SiegeEngine implements Cloneable {
 
 	// Core variables
-	public SiegeEngineType type;
-	public Boolean enabled;
-	public String id;
-	public String itemName;
-	public List<String> itemLore;
-	public String worldName;
-	public UUID entityId;
-    public Entity entity;
-    public int yOffset;
-    public int xOffset;
-    public int pitchOffset;
-    public int maxFuel;
-    public double placementOffsetY;
-    public float velocityPerFuel;
+	private SiegeEngineType type;
+	private Boolean enabled;
+	private String id;
+	private String itemName;
+	private List<String> itemLore;
+	private String worldName;
+	private UUID entityId;
+    private Entity entity;
+    private int yOffset;
+    private int xOffset;
+    // private int pitchOffset; // Internal - Never Initialized and Defaulted to 0
+    private int maxFuel;
+    private double placementOffsetY;
+    private float velocityPerFuel;
 	
 	// Firing variables
-    public int shotAmount;
-    public long nextShotTime;
-    public int millisecondsBetweenFiringStages;
-    public int millisecondsBetweenReloadingStages;
-    public int millisecondsToLoad;
-    public int modelNumberToFireAt;
-    public boolean cycleThroughModelsWhileFiring;
-    public boolean setModelNumberWhenFullyLoaded;
-    public boolean rotateSideways;
-    public boolean rotateUpDown;
-    public boolean rotateStandHead;
-    public boolean hasFired;
-    public boolean hasReloaded;
-    public int taskNumber;
-    public List<Integer> firingModelNumbers;
-    public int readyModelNumber;
-    public int nextModelNumber;
-    public int preFireModelNumber;
-    public SiegeEngineAmmoHolder ammoHolder;
+    private int shotAmount;
+    private long nextShotTime; // Internal
```

Comment:
@ShermansWorld I added // Internal next to anything that was just used within the class and never accessed outside of it. Is there any standard for this or is there any way you want these variables to be grouped? I can also just remove the comments and leave them in their current places if it isn't relevant info.